### PR TITLE
Loops: while and for

### PR DESCRIPTION
**Pull request by alvar-o**
I had to delete the previous PR because of an issue with author email in Git. Pulling it again.

### CODE REVIEW

**Comment by alvar-o** [comment]
Hey, I need some guidance here: I signed the CLA and added my email address to my Git and Github account. Still the commit was made prior to that and as such it doesn't have the same signature. Should I delete this and create a new commit with new files?

**Comment by Dheyson** [comment]
Is your email address the same used by your Github login?
You can [re-check here](https://github.com/javascript-tutorial/pt.javascript.info/pull/51), as soon that you have already signed.
I can`t say more after this. If still persist, you might try close this PR and open another one.

File: 1-js/02-first-steps/12-while-for/1-loop-last-value/solution.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ while (i) {
 }
 ```
 
-Every loop iteration decreases `i` by `1`. The check `while(i)` stops the loop when `i = 0`.
+Cada iteração do loop subtrai `1` de `i`. O teste `while(i)` interrompe o loop quando `i = 0`.
```
Comment (Dheyson) [inline 1-js/02-first-steps/12-while-for/1-loop-last-value/solution.md]: Don`t know if `subtrai` would be the best approach, considering the way you conjugate the _decrease_ word in the following sentence. I mean, you use `decresce`. Just a suggestion, but the translation its understandable. I think another person should check this.

File: 1-js/02-first-steps/12-while-for/2-which-value-while/task.md [hunk 2]
```diff
@@ -2,19 +2,19 @@ importance: 4
 
 ---
 
-# Which values does the while loop show?
+# Quais valores o loop while retorna?
 
-For every loop iteration, write down which value it outputs and then compare it with the solution.
+Para cada iteração dos loops, anote o valor que ele retorna, depois compare com a solução.
 
-Both loops `alert` the same values, or not?
```
Comment (Dheyson) [inline 1-js/02-first-steps/12-while-for/2-which-value-while/task.md]: I think would be a good a idea to ask to add the word return here. As you translated.

File: 1-js/02-first-steps/12-while-for/7-list-primes/task.md [hunk 3]
```diff
@@ -2,16 +2,16 @@ importance: 3
 
 ---
 
-# Output prime numbers
+# Retornar número primos
 
-An integer number greater than `1` is called a [prime](https://en.wikipedia.org/wiki/Prime_number) if it cannot be divided without a remainder by anything except `1` and itself.
+Um número inteiro maior que `1` é chamado de [primo](https://pt.wikipedia.org/wiki/Número_primo) se ele não puder ser divido por nada exceto por `1` e por si mesmo sem gerar um resto.
```
Comment (Dheyson) [inline 1-js/02-first-steps/12-while-for/7-list-primes/task.md]: Dividido por nada[...]

**Comment by alvar-o** [comment]
> Just that situation that I overview

Thank you for your considerations. I will review the changes you suggested, but will be doing that in a new PR, since this one is not working. :)

File: 1-js/02-first-steps/12-while-for/1-loop-last-value/solution.md (hunk 1 above)
Comment (alvar-o) [reply 1-js/02-first-steps/12-while-for/1-loop-last-value/solution.md]: I understand. I suppose we could say 

> "Cada iteração do loop decresce `i` em `1`."

I find the phrasing with "subtrai" to sound more natural to Brazilian speakers, but I'll change it.

File: 1-js/02-first-steps/12-while-for/7-list-primes/task.md (hunk 3 above)
Comment (alvar-o) [reply 1-js/02-first-steps/12-while-for/7-list-primes/task.md]: I'll  rephrase that as:

> Um número inteiro maior que `1` é chamado de [primo](https://pt.wikipedia.org/wiki/Número_primo) se ele não puder ser divido por **nenhum outro número** exceto por `1` e por si mesmo sem gerar um resto.

**Comment by alvar-o** [comment]
It took me longer than I expected to come back to this. I am going to close this PR and open a new one with the changes.

**Pull request closed without merging**